Sources/App/Core/SiteURL+siteMap.swift

```diff
@@ -54,6 +54,8 @@ extension SiteURL {
                 return .hourly
             case .rssReleases:
                 return .hourly
+            case .search:
+                return .hourly
```

Comment:
It doesn't matter very much, but I'm not sure `hourly` is correct here. I was going to suggest `never` as the search page on its own is not something we'd want Google sending people to the search page directly, but the [docs](https://www.sitemaps.org/protocol.html) for `never` say:

> The value "never" should be used to describe archived URLs.

Which isn't true.

It's probably fine as `hourly`, but I thought I'd bring it up.

Reply:
👍  I don't really know what to pick here, I just filled in what I thought the change rate was going to be.

Reply:
It's just telling Google how often to come back and re-index that page., which should really be `never` as there will never be anything on that page for Google to index. It's not a huge deal